.prettierignore

```diff
@@ -1 +1 @@
-schemas
+./schemas
```

Comment:
With the recent repo merge, the typescript source now has a `schemas` directory as well